frontend/shared/components/src/groups/components/ReduceablePriceInput.vue

```diff
@@ -19,55 +25,94 @@
 
 <script setup lang="ts">
 import { SimpleError } from '@simonbackx/simple-errors';
-import { PriceInput, STErrorsDefault, useErrors, useValidation, Validator } from '@stamhoofd/components';
+import { PriceInput, STErrorsDefault, useErrors, useOrganization, usePlatform, useValidation, Validator } from '@stamhoofd/components';
 import { Group, ReduceablePrice } from '@stamhoofd/structures';
+import { Formatter } from '@stamhoofd/utility';
 import { computed } from 'vue';
 import { ErrorBox } from '../../errors/ErrorBox';
 import { useFinancialSupportSettings } from '../hooks';
 
 const props = withDefaults(
     defineProps<{
-        validator: Validator,
-        errorBox?: ErrorBox|null,
-        min?: number|null,
-        group?: Group|null,
-        title?: string
+        validator: Validator;
+        errorBox?: ErrorBox | null;
+        min?: number | null;
+        group?: Group | null;
+        title?: string;
+        defaultMembershipTypeId?: string | null;
     }>(),
     {
         errorBox: null,
         min: null,
         title: 'Prijs',
-        group: null
-    }
+        group: null,
+        defaultMembershipTypeId: null,
+    },
 );
-const model = defineModel<ReduceablePrice>({ required: true })
+const model = defineModel<ReduceablePrice>({ required: true });
 const ownErrors = useErrors();
-const {enabled, financialSupportSettings} = useFinancialSupportSettings({
+const { enabled, financialSupportSettings } = useFinancialSupportSettings({
     group: computed(() => props.group),
-})
+});
+const platform = usePlatform();
+const organization = useOrganization();
+
+const $showReducedPrice = computed(() => enabled || reducedPrice.value !== null);
+
+const defaultReducedPrice = computed(() => getDefaultPrice(true));
+const defaultPrice = computed(() => getDefaultPrice(false));
+const minPriceDifference = computed(() => defaultPrice.value - defaultReducedPrice.value);
+
+function getDefaultPrice(isReduced: boolean) {
+    if (!props.defaultMembershipTypeId) {
+        return 0;
+    }
+
+    const defaultMembershipTypeId = props.defaultMembershipTypeId;
+    const defaultMembership = platform.value.config.membershipTypes.find(mt => mt.id === defaultMembershipTypeId);
+    if (!defaultMembership) {
+        return 0;
+    }
 
-const $showReducedPrice = computed(() => enabled || reducedPrice.value !== null)
+    return defaultMembership.getPrice(platform.value.period.id, new Date(), organization.value?.meta.tags ?? [], isReduced) ?? 0;
+}
+
+function formatPriceForPlatform(price: number) {
+    return `Je betaalt ${Formatter.price(price)} aan KSA Nationaal.`;
```

Comment:
Kan je hier specifiek nog een vertaling toevoegen, anders gaan we dit vergeten aan te passen